web/src/components/planner/Planner.vue

```diff
@@ -255,6 +256,11 @@
     return product ?? byProduct ?? undefined
   }
 
+  const getPart = (factory: Factory, partId: string): PartMetrics | undefined => {
```

Comment:
This is fairly pointless, as you can just reference the part object out of the factory directly. It offers nothing of real value.

Reply:
Sure, I can remove this. I was just following the pattern of getting a Product since they are similar. But I'm all for deleting it 👍

Reply:
Removed in latest commit